attributes/default.rb

```diff
@@ -67,6 +67,7 @@
 
 default['hopsworks']['staging_dir']              = node['hopsworks']['dir'] + "/staging"
 default['hopsworks']['conda_cache']              = node['hopsworks']['staging_dir'] + "/glassfish_conda_cache"
+default['hopsworks']['upload_staging_dir']       = "Projects"
```

Comment:
Why do we need a variable to say that the chunks are going to be placed into the project?